### ISSUE TITLE
Might PubLayNet label mapping be incorrect?

### ISSUE THREAD

**Issue by opyate** [Layout-Parser/layout-parser#178]
Hello, thanks for this very useful toolkit.

According to the [official PubLayNet docs](https://github.com/ibm-aur-nlp/PubLayNet/tree/master/pre-trained-models#convert-pre-trained-model-for-fine-tuning-on-another-target-dataset), they had this mapping:

| Category id | Label |
| :---: | :--- |
| 0 | Background |
| 1 | Text |
| 2 | Title |
| 3 | List |
| 4 | Table |
| 5 | Figure |

But in the [model zoo docs](https://github.com/Layout-Parser/layout-parser/blob/main/docs/notes/modelzoo.md#model-label_map), you specify this mapping:

```python
{0: "Text", 1: "Title", 2: "List", 3:"Table", 4:"Figure"}
```

Please advise on the discrepancy. 
Kind regards!
